CalendarPicker/Weekdays.js

```diff
@@ -12,28 +12,29 @@ export default function Weekdays(props) {
     startFromMonday,
     weekdays,
     textStyle,
-    dayLabelsCustomWrapper,
-    isChangeSundayColor,
-    sundayColor,
+    dayLabelsWrapper,
+    weekdayStyles,
   } = props;
   let wd = weekdays;
   if (!wd) {
     wd = startFromMonday? Utils.WEEKDAYS_MON : Utils.WEEKDAYS; // English Week days Array
   }
 
   return (
-    <View style={[styles.dayLabelsWrapper, dayLabelsCustomWrapper]}>
+    <View style={[styles.dayLabelsWrapper, dayLabelsWrapper]}>
       { wd.map((day, key) => {
         let updatedStyle = textStyle;
-        if (isChangeSundayColor) {
-          let matchIndex = 0;
-          if (startFromMonday) {
-            matchIndex = 6;
-          }
-          if (key === matchIndex) {
-            updatedStyle = {...updatedStyle, ...{color: sundayColor}};
+        try {
+          if (weekdayStyles.length > 0) {
```

Comment:
If you change this to
```
if (weekdayStyles[+key]) {
```
It will be able to support either an array or object because ES6 supports access of variable key names using the [key] syntax.

Reply:
OK. I will do this and other changes.